src/components/Project/index.jsx

```diff
@@ -10,75 +10,86 @@ import Grade from './Grade';
-    variables: { email: data ? data.me.email : '' },
-  });
+	const [ result ] = useQuery({
+		query: userQuery,
+		variables: { email: data ? data.me.email : '' }
+	});
 
-  useEffect(() => {
-    if (result.data) {
-      setUser(result.data.person.role.privateNote);
-    }
-  }, [result.data]);
+	useEffect(
+		() => {
+			if (result.data) {
+				setUser(result.data.person.role.privateNote);
+			}
+		},
+		[ result.data ]
+	);
 
-  if (fetching){
-    return <p>Loading...</p>
-  } else if (error){
-    return <p>Error "PROJECT_VIEW_QUERY": {error.name} {error.message}</p>
-  } else if (data){
-    return (
-      <div className={parentProjectContainer}>
-        <div className={projectPageContents}>
-          <div>
-            <Header projectId={id} />
-          </div>
-          <div className={projectContainer}>
-            <div className={editorFeedContainer}>
-              <h2>Repos Code Health</h2>
-              <GitHubRepos />
-              <div className={gradeContainer}>
-                <Grade ccrepos={data.project.product.grades} />
-              </div>
-              <h2>Project Notes</h2>
-              {user === true ? (
-                <NoteEditor
-                  executeQuery={executeQuery}
-                  user={data.me}
-                  projectId={id}
-                  projectManagers={data.project.projectManagers}
-                />
-              ) : null}
-  
-              <NotesFeed projectId={id} privateBol={user} />
-            </div>
-            <div className={teamContainer}>
-              <Team projectId={id} />
-            </div>
-          </div>
-        </div>
-      </div>
-    )
-  } else {
-    return <p>End of line.</p>
-  }
+	if (fetching && !data ) {
+		return <p>Please Wait... Loading...</p>;
+	} else if (error) {
+		return (
+			<p>
+				Error "PROJECT_VIEW_QUERY": {error.name} {error.message}
+			</p>
+		);
+	} else if (data) {
+		return (
+			<div className={parentProjectContainer}>
+				<div className={projectPageContents}>
+					<div>
+						<Header projectId={id} />
+					</div>
+					<div className={projectContainer}>
+						<div className={editorFeedContainer}>
+							<h2>Repository Statistics</h2>
+							<GitHubRepos
+								ghrepos={data.project.product.GHRepos}
+								productId={data.project.product.id}
+								executeQuery2={executeQuery}
+							/>
+							<div className={gradeContainer}>
+								<Grade ghrepos={data.project.product.grades} />
+							</div>
+							<h2>Project Notes</h2>
+							{user === true ? (
+								<NoteEditor
+									executeQuery={executeQuery}
+									user={data.me}
+									projectId={id}
+									projectManagers={data.project.projectManagers}
+								/>
+							) : null}
+
+							<NotesFeed projectId={id} privateBol={user} />
+						</div>
+						<div className={teamContainer}>
+							<Team projectId={id} />
+						</div>
+					</div>
+				</div>
+			</div>
+		);
+	} else {
+		return <p>End of line.</p>;
+	}
```

Comment:
Curious to why it's formatting like this....creates extra work when we merge.

Reply:
Seems to be an issue with Prettier. I noticed this with other commits:

https://github.com/Lambda-School-Labs/mission-control-fe/pull/239/commits/3a1244bed89cc9d81ea2d46941c53a79d08935a4

Reply:
I use prettier, do we have a linter for the front end? Not sure why it looks like this but I would have to say prettier is doing it.

Reply:
Hmmm!

looks like there is a linter on front end

` "extends": ["airbnb", "plugin:prettier/recommended", "prettier/react"],`

and in .prettierrc

`{
  "printWidth": 80,
  "singleQuote": true,
  "trailingComma": "es5"
}
`